### PR TITLE
Anti-censorship proposal

### PR DESCRIPTION
**Pull request by Snawoot**
This proposal is intended to sketch whole picture. I expect that specific details will be added in separate documents following this general proposal.

### CODE REVIEW

File: general/StateCensorshipCircumvention_2021115.MD [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+
+Consider following situation: circumvention service operator wants to provide services to user, but censor wants to prevent it. Circumvention service operator releases application with some non-trivial logic to discover working relays and let users to pass their traffic through. All what censor needs to do is just use that application and keep track on IP addresses used to forward actual traffic. This way censor may ban all IP addresses learned until application will cease to work.[^2]
+
+The ultimate task is to find a solution for meta-data obfuscation satisfying all requirements presented above.
+
+## Related work
+
+### Academic sources
+
+Actual work doesn't claims novelty and for this reason author skips detailed comparison against prior art to outline own contribution.
```
Comment (mdomasevicius) [inline general/StateCensorshipCircumvention_2021115.MD]:
```suggestion
Actual work doesn't claim novelty and for this reason the author skips detailed comparison against prior art to outline own contribution.
```

Comment (Snawoot) [reply general/StateCensorshipCircumvention_2021115.MD]: Thanks!

**Comment by Zensey** [comment]
An update on Runet practices

https://ripe83.ripe.net/archive/video/alexander-isavnin_sovereign-internet-and-the-number-resources_main-20211122-143204.mp4

**Review by tomasmik — APPROVED** [review]

**Review by chompomonim — APPROVED** [review]

**Review by mdomasevicius — APPROVED** [review]
I am wondering if this proposals should have an appendix that JUST consolidates all of implications it potentially has for both providers and consumers in relation to current workflow and UX. Historically there have been "WTFs" after certain feature were implemented by @chompomonim  himself so for less technical people like our support staff it would also give a chance to raise their concerns here - after all they have most exposure to the users and their day to day problems.

**Pull request merged**